tests/io/test_docker.py

```diff
@@ -11,7 +11,7 @@
 from ..utils import test_output_dir
 
 
-@pytest.mark.skip  # skipped as long as its imcomplete
+# @pytest.mark.skip  # skipped as long as its imcomplete
```

Comment:
Should this line be removed entirely? And perhaps also the commented out code below?

Reply:
If it runs, sure! It didn't run before, hence why I skipped it.

Reply:
I've removed the commented skip in 6c375c4.